composer.json

```diff
@@ -1,5 +1,6 @@
 {
     "name": "paysera/lib-dependency-injection",
+    "version": "1.3.1",
```

Comment:
It is recommended to omit the `version` property if the
package repository can infer the version from a tag, for example.

More on that can be found in [Composer documentation](https://getcomposer.org/doc/04-schema.md#version).